src/data/markdown/translated-guides/en/02 Using k6/05 k6 Options/02 Reference.md

```diff
@@ -444,14 +446,14 @@ export const options = {
   hosts: {
     'test.k6.io': '1.2.3.4',
```

Comment:
```suggestion
```

Should we delete this line? Doesn't the wildcard below already do this redirect?

Reply:
It does, but I think it's important to document all ways the feature can be used, and we should keep this most basic way of doing the mapping. The example itself doesn't have to represent a real-world usage scenario. We could always use a different domain instead, maybe `*.grafana.com`, but I think this is fine.

Reply:
>We could always use a different domain instead, maybe *.grafana.com, but I think this is fine.
 
I'd rather do this. Or `test-api.k6.io` or whatever. As it is right now, that line in the script doesn't really do anything. So I think it creates ambiguity. Readers will look at it and wonder, _wait, doesn't the wildcard already include that domain? Or do I not understand how this works?_ That was my thought process, at least.

Reply:
OK. Done in 934582db.